packages/contracts-bedrock/scripts/fpac/FPACOPS.s.sol

```diff
@@ -78,6 +83,38 @@ contract FPACOPS is Deploy, StdAssertions {
         );
     }
 
+    function initializeAnchorStateRegistryProxy() internal broadcast {
+        console.log("Initializing AnchorStateRegistryProxy with AnchorStateRegistry.");
+
+        AnchorStateRegistry.StartingAnchorRoot[] memory roots = new AnchorStateRegistry.StartingAnchorRoot[](3);
+        roots[0] = AnchorStateRegistry.StartingAnchorRoot({
+            gameType: GameTypes.CANNON,
+            outputRoot: OutputRoot({
+                root: Hash.wrap(cfg.faultGameGenesisOutputRoot()),
+                l2BlockNumber: cfg.faultGameGenesisBlock()
+            })
+        });
+        roots[1] = AnchorStateRegistry.StartingAnchorRoot({
+            gameType: GameTypes.PERMISSIONED_CANNON,
+            outputRoot: OutputRoot({
+                root: Hash.wrap(cfg.faultGameGenesisOutputRoot()),
+                l2BlockNumber: cfg.faultGameGenesisBlock()
+            })
+        });
+        roots[2] = AnchorStateRegistry.StartingAnchorRoot({
```

Comment:
This script is for live deployments, can remove the alphabet root

Reply:
Done